labelbox/schema/data_row.py

```diff
@@ -185,6 +182,9 @@ def export_v2(client: 'Client',
         >>>     task.result
         """
 
+        if not data_rows and not global_keys:
```

Comment:
Let's add an error here if both are provided, as well

Reply:
Would it be better to place this validation inside `build_filters` then, so we have one place where these parameters are checked?

Reply:
I can't do it easily... ping me if you want to discuss more

Reply:
that is equivalent to my comment `I can't do it easily... ` so yes I second this and actually I could do it easily, but more work than allocated for this PR. However it would be nice to keep this part of code orderly for future changes

Reply:
There was a decision made to use TypedDict, because our product teams really dislike making customers import additional things. I agree on that point. Using Pydantic or python classes will require customers to import those, and instantiate those classes. TypedDict allows them to simply pass in a dictionary.